Disable error on warnings.
rustc 1.82.0 introduces a new deprecation and the warning is treat as error.

Index: src/main.rs
--- src/main.rs.orig
+++ src/main.rs
@@ -2,8 +2,6 @@
 // <https://rust-lang.github.io/rust-clippy/master/index.html>
 #![allow(clippy::needless_return)]
 //
-// Fail build on Clippy warnings
-#![deny(warnings)]
 
 #[macro_use]
 extern crate lazy_static;
